infrastructure/main.tf

```diff
@@ -38,9 +49,9 @@ module "citizen-frontend" {
   location = "${var.location}"
   env = "${var.env}"
   ilbIp = "${var.ilbIp}"
-  is_frontend  = true
+  is_frontend = "${var.env != "preview" ? 1: 0}"
```

Comment:
Why not `true : false`?

Reply:
doesn't work :( Get a boolean conversion error

Reply:
hah not quite that scary, but similar, Boolean isn't a supported data type in terraform it only kind of works